agent/apps/files/app_code/views.py

```
@@ -0,0 +1,87 @@
+"""
+ویوهای اپلیکیشن فایل‌ها
+Application Views for Files
+"""
+
+from typing import Any, Dict
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)